docs/application/vstools/Tizen/hybrid.md

```diff
@@ -0,0 +1,81 @@
+# Hybrid application development
+
+
+## Develop application
+
+The following sections explain how to use Visual Studio Extension for Tizen to develop your hybrid applications. Hybrid application for Tizen supports to have different types(.NET/Native/Web) of project in same Solution/Workspace.
+
+
+### Create Tizen .Net project
+
+To create a Tizen .Net project: Refer to [Create Tizen .Net project](dotnet.md)
+
+
+### Add Tizen Native project
+
+To add a Tizen Native project into the above created .NET project to make the Hybrid solution:
+
+1. In the Visual Studio Solution Explorer, Right Click on Solution **Add &gt; New Project**.
+
+2. For creating Native application, select **C++** and **Tizen** options in the dropdown list. Then, select **Tizen Native project** and click **Next**.
+
+   ![Create Tizen project](media/native_create_project.PNG)
+
+3. In the configure window, type the name for your project and click **Create**.
+
+   ![Configure project](media/native_sub_project_configure.PNG)
+
+4. In the **Tizen Profile Select** window, select the required template for your project, then click **OK**.
+   
+   ![Version selection](media/native_platform_version.PNG)
+
+5. The visual studio window with newly created project appears on the **Solution Explorer**.
+
+   ![Visual Studio screen](media/hybrid_vs_screen.PNG)
+
+
+### Set Startup Project
+
+1.  From **Solution Explorer** Right click on the desired Project and select **Set as Startup Project**.
+
+
+### Add Project Dependency
```

Comment:
Headings should follow sentence style case.
Ref: https://github.com/Samsung/tizen-docs/blob/master/reviewguide/review_points_guide.md#headings
```suggestion
### Add project dependency
```